server/models/User.js

```diff
@@ -0,0 +1,13 @@
+const mongoose = require("mongoose");
```

Comment:
we're kinda mixing import styles here, kinda my fault I was in the process of changing it. So there's 2 ways of importing, either using require or using something called ES modules. We should use ES modules it's more straightforward.

So instead of this it would be: 
`import mongoose from "mongoose";`